e2e/windows/scenarios/windows/property-windows-template.json

```diff
@@ -92,7 +92,7 @@
     "--cloud-provider": "external",
     "--enforce-node-allocatable": "\"\"\"\"",
     "--eviction-hard": "\"\"\"\"",
-    "--feature-gates": "CSIMigration=true,CSIMigrationAzureDisk=true,CSIMigrationAzureFile=true,DelegateFSGroupToCSIDriver=true,DynamicKubeletConfig=false",
```

Comment:
Will it fail the tests with k8s < v1.27?

Reply:
It will not. I've tried 1.24.10, 1.25.6 and attached pipelines in the channel.